mne/decoding/transformer.py

```diff
@@ -147,13 +147,13 @@ def inverse_transform(self, epochs_data, y=None):
         return X
 
 
+@deprecated("Class 'ConcatenateChannels' has been renamed to "
+            "'EpochsVectorizer' and will be removed in release 0.11.")
 class ConcatenateChannels(TransformerMixin):
     """Concatenates data from different channels into a single feature vector
 
     Parameters
     ----------
-    info : instance of Info
-        The measurement info.
```

Comment:
it didn't pass doc tests because of deprecated.
I found that removing parameters doc fix this issue